metadata-models/src/main/pegasus/com/linkedin/dashboard/DashboardInfo.pdl

```diff
@@ -0,0 +1,42 @@
+namespace com.linkedin.dashboard
+
+import com.linkedin.common.AccessLevel
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.ChartUrn
+import com.linkedin.common.Time
+
+/**
+ * Information about a dashboard
+ */
+record DashboardInfo {
+
+  /**
+   * Title of the dashboard
+   */
+  title: string
+
+  /**
+   * Detailed description about the dashboard
+   */
+  description: string
+
+  /**
+   * Charts in a dashboard
+   */
+  charts: array[ChartUrn] = [ ]
```

Comment:
why did you choose to have a default for this array vs. making this an optional field?

Reply:
Dashboard without a chart doesn't make any sense in general.